diff --git a/build/index.js b/build/index.js
index ada7cdcac9d6893879753f769fb07a8e234fdc15..5af82ef76a14eb10b19bb2c4d6c5f50a7ada8adb 100644
--- a/build/index.js
+++ b/build/index.js
@@ -49,6 +49,7 @@ function TextInput({ value: originalValue, placeholder = '', focus = true, mask,
         if (key.upArrow ||
             key.downArrow ||
             (key.ctrl && input === 'c') ||
+            (key.ctrl && /^[a-z]$/i.test(input)) ||
             key.tab ||
             (key.shift && key.tab)) {
             return;
